packages/sanity/src/media-library/plugin/VideoInput/VideoPreview.tsx

```diff
@@ -39,6 +41,9 @@ export function VideoPreview(props: VideoAssetProps) {
   const asset = value?.asset
   const sourcesFromSchema = schemaType.options?.sources
   const accept = get(schemaType, 'options.accept', '')
+  const isStaging = useClient({apiVersion: DEFAULT_API_VERSION})
+    .config()
+    .apiHost.endsWith('.sanity.work')
```

Comment:
probably better to do:
```suggestion
	// @ts-expect-error: __SANITY_STAGING__ is a global env variable set by the vite config
  const isStaging = typeof __SANITY_STAGING__ !== 'undefined' && __SANITY_STAGING__ === true
```